galasa-ui/src/styles/test-runs/saved-queries/QueryItem.module.css

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+.sideNavItem {
+  display: flex;
+  gap: 0.5rem;
+  align-items: flex-start;
+  padding: 0.5rem 0;
+  position: relative;
+  transition: background-color 0.15s ease-in-out;
+}
+
+.sideNavItem.collapsed {
+  display: none !important;
```

Comment:
`!important` isn't great to use since it forcefully overrides values for CSS properties (and in turn messes with specificity) and can make debugging CSS really challenging, so it should be the last thing that you would want to use.

Could `collapsed` be a CSS state? Then it could be used like `.sideNavItem:state(collapsed)`?